### PR TITLE
[api] Fix return type of WoWs clans info

### PR DESCRIPTION
**Pull request by kakwa**
clans info can return several clans in the form of a map `{<clan_id>: {<DATA>}`.

Example:

```
https://api.worldofwarships.eu/wows/clans/info/?application_id=<app_id>&clan_id=500139805%2C500140039&fields=name
```

```json
{
    "status": "ok",
    "meta": {
        "count": 2
    },
    "data": {
        "500139805": {
            "name": "PONYHOF - Life is not all guns and roses."
        },
        "500140039": {
            "name": "TheDrunkenSunken"
        }
    }

}
```

* added a patch case in tools/generator/internal/patches.go
* commit the updated wargaming/wows_clans_info.go (map[int]*wows.ClansInfo instead of *wows.ClansInfo for the return)

note: all the files got non-significant changes like:
```diff
-//     Valid realms: RealmAsia, RealmEu, RealmNa
+//
+//     Valid realms: RealmAsia, RealmEu, RealmNa
```

But I've not commited them (to not clutter the commit)

### CODE REVIEW

**Comment by IceflowRE** [comment]
The extra line of slashes is not intended, could you remove it? And change the commit title to `[api] Fix return type of WoWs clans info`. Thanks!

File: wargaming/wows_clans_info.go [hunk 1]
```diff
@@ -14,10 +14,13 @@ import (
...
-//     Valid realms: RealmAsia, RealmEu, RealmNa
+//
+//	Valid realms: RealmAsia, RealmEu, RealmNa
+//
 // clanId:
-//     Clan ID. Maximum limit: 100. Min value is 1.
-func (service *WowsService) ClansInfo(ctx context.Context, realm Realm, clanId []int, options *wows.ClansInfoOptions) (*wows.ClansInfo, error) {
+//
+//	Clan ID. Maximum limit: 100. Min value is 1.
+func (service *WowsService) ClansInfo(ctx context.Context, realm Realm, clanId []int, options *wows.ClansInfoOptions) (map[int]*wows.ClansInfo, error) {
```
Comment (IceflowRE) [inline wargaming/wows_clans_info.go:23]: This change is not intended.

File: wargaming/wows_clans_info.go [hunk 2]
```diff
@@ -14,10 +14,13 @@ import (
...
 // realm:
-//     Valid realms: RealmAsia, RealmEu, RealmNa
+//
+//	Valid realms: RealmAsia, RealmEu, RealmNa
+//
 // clanId:
-//     Clan ID. Maximum limit: 100. Min value is 1.
-func (service *WowsService) ClansInfo(ctx context.Context, realm Realm, clanId []int, options *wows.ClansInfoOptions) (*wows.ClansInfo, error) {
+//
+//	Clan ID. Maximum limit: 100. Min value is 1.
```
Comment (IceflowRE) [inline wargaming/wows_clans_info.go]: This change is not intended.

Comment (kakwa) [reply wargaming/wows_clans_info.go]: I've cleaned it up but it was coming from the generator, it is not something I've introduced.

Maybe it's a different behavior due to platform differences?

(Debian Sid + go version go1.19.4 linux/amd64 in my case)

**Comment by kakwa** [comment]
I will also probably push another similar fix for Account Info (also in wows).

And I'm not sure, but I think there is an issue in the Clan Info payload as well (Members is once again a map[int])

**Pull request merged**